current_spring2026/graph/neo4j_client.py

```
@@ -0,0 +1,102 @@
+}
+"""
+
+
+def create_schema():
+    print("Creating Neo4j schema...")
+    with get_session() as session:
+        for query in SCHEMA_QUERIES:
+            try:
+                session.run(query)
+                print(f"  OK: {query[:60]}...")
+            except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)